packages/aws-cdk/lib/api/bootstrap/bootstrap-template.json

```diff
@@ -338,8 +338,9 @@
               "Statement": [
                 {
                   "Action": [
-                    "cloudformation:CreateChangeSet",   "cloudformation:DeleteChangeSet",
-                    "cloudformation:DescribeChangeSet", "cloudformation:DescribeStacks",
+                    "cloudformation:DescribeStackEvents", "cloudformation:GetTemplate",
```

Comment:
Didn't we already add these?

There is a statement called CliPermissions, I'd rather you add them there (I want to start annotating this giant permissions set with why we need them, because it's growing big).

Reply:
> Didn't we already add these?

No. Searching the [current `bootstrap-template.json` on master](https://github.com/aws/aws-cdk/blob/c63db52c94b5194974f4f38bf820e79ded89001d/packages/aws-cdk/lib/api/bootstrap/bootstrap-template.json#L1) does not give any results for `GetTemplate`.

> There is a statement called CliPermissions, I'd rather you add them there (I want to start annotating this giant permissions set with why we need them, because it's growing big).

I don't see a statement called like that [anywhere in the current file](https://github.com/aws/aws-cdk/blob/c63db52c94b5194974f4f38bf820e79ded89001d/packages/aws-cdk/lib/api/bootstrap/bootstrap-template.json#L1).

Reply:
Doh! Misremembering where we put this:

https://github.com/aws/aws-cdk/blob/feat/convmode/packages/aws-cdk/lib/api/bootstrap/bootstrap-template.json#L353-L363

Reply:
Changed in the newest revision to what you have in that branch.

BTW, do we really need `DeleteStack` on `"*"`? That seems a little wide...